frontend/src/AlertsManagement/AlertInfo.js

```diff
@@ -46,6 +47,53 @@ class AlertInfo extends Component {
     newAlert.status = statusToChangeTo;
     this.setState({ alert: newAlert });
   }
+
+  createLineChart = (visData, index) => {
+    const color = visData.relatedDataList ? 'rgba(243, 0, 87, 1)' : 'rgba(63, 81, 181, 1)';
+    const chartData = {
+      labels: [ ...visData.dataPoints.keys() ],
+      datasets: [
+        {
+          label: visData.metricName + ' for ' + visData.dimensionName,
+          fill: false,
+          lineTension: 0.1,
+          backgroundColor: 'rgba(75,192,192,0.4)',
+          borderColor: color,
+          borderCapStyle: 'butt',
+          borderDash: [],
+          borderDashOffset: 0.0,
+          borderJoinStyle: 'miter',
+          pointBorderColor: color,
+          pointBackgroundColor: '#fff',
+          pointBorderWidth: 1,
+          pointHoverRadius: 5,
+          pointHoverBackgroundColor: color,
+          pointHoverBorderColor: 'rgba(220,220,220,1)',
+          pointHoverBorderWidth: 2,
+          pointRadius: 1,
+          pointHitRadius: 10,
+          data: [ ...visData.dataPoints.values() ]
+        }
+      ]
+    };
+
+    return <Line key={index} data={chartData} />;
+  }
+
+  visualizeRelatedData = (anomaly) => {
+    let relatedDataCharts = [];
+    anomaly.relatedDataList.forEach((relatedData, index) => 
+      relatedDataCharts.push(
+        <Fragment key={index}>
+          <Typography variant="body2" style={{ margin: '10px' }}>
```

Comment:
i think you have code that fixes the related data charts so that they align with the anomaly data rt?
is this the updated code for it?

Reply:
nah I'm going to make a new PR for UI adjustments